uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/LeavePeriodEntitlementTest.php

```diff
@@ -423,6 +423,49 @@ public function testGetReturnsOnlyDataLinkedToContactsThatLoggedInUserManagesWhe
     $this->assertEquals($contact2['id'], $result['values'][0]['contact_id']);
   }
 
+  public function testGetReturnsOnlyDataLinkedToContactsThatLoggedInUserManagesWhenLoggedInIsALeaveApproverWithOneOfTheAvailableRelationships() {
+    $this->setLeaveApproverRelationshipTypes([
+      'approves leaves for',
+      'manages things for',
+    ]);
+
+    $manager1 = ContactFabricator::fabricate();
+    $manager2 = ContactFabricator::fabricate();
+    $contact1 = ContactFabricator::fabricate();
+    $contact2 = ContactFabricator::fabricate();
+
+    $this->setContactAsLeaveApproverOf($manager1, $contact2, null, null, true, 'manage things for');
```

Comment:
Is it possible to also add a test where a manager manages two contacts but with different relationships?
i.e manages contactA with 'approves leaves for' and contactB with 'manages things for'